docs/how-to/h-use-spark-client-from-python.md

```diff
@@ -121,8 +127,8 @@ registry.delete(service_account.id)
 
 #### Manage Primary Accounts
 
-In the Spark Client Snap, there is the notion of the primary service account, the 
-one that would be chosed by default, if no specific information is provided. The
+Spark has a notation of the so called 'primary' service account, the 
```

Comment:
spark8t has this notion, spark doesn't afaik

Reply:
Revised the text!